setup.py

```diff
@@ -68,8 +68,12 @@
     ],
     extras_require={
         'rllib': [
+            'GPUtil',
```

Comment:
I think these GPU deps aren't needed for the rllib example per-se? Just for this devcontainer? Perhaps add them in the Dockerfile?

Reply:
Not everyone will be running GPU though, so I don't think these deps are always needed? Since this is platform dependent I suggest that this is added to the `Dockerfile` or perhaps to a `[gpu]` target if it seems likely that others will want these packages.

Reply:
Sure I'll split these out